web/src/list_widget.ts

```diff
@@ -540,11 +548,11 @@ export function create<Key = unknown, Item = Key>(
     return widget;
 }
 
-export function get(name) {
-    return DEFAULTS.instances.get(name) || false;
+export function get(name: string): ListWidget | false {
+    return DEFAULTS.instances.get(name) ?? false;
```

Comment:
Does this need to return `false` as a fallback? Might be worth a follow-up PR as a quick refactoring project.

Reply:
Yeah, I am planning to refactor this module a bit after it gets translated to TypeScript so that I will know I am not breaking
things when doing refactoring due to all those nice typescript errors. So yeah I will do a follow-up refactoring project :)